src/scala/microsoft-spark-2.4.x/src/main/scala/org/apache/spark/api/dotnet/SerDe.scala

```diff
@@ -138,6 +139,11 @@ object SerDe {
     (0 until len).map(_ => readRow(in)).toList.asJava
   }
 
+  def readAnyArr(in: DataInputStream): Seq[Any] = {
+    val len = readInt(in)
+    (0 until len).map(_ => readObject(in).asInstanceOf[Any])
```

Comment:
is `.asInstanceOf[Any]` needed?    Aren't all types `Any` in `scala` ?

Reply:
True. Changed, thanks!